benches/static_and_dynamic_functions.rs

```diff
@@ -146,42 +200,52 @@ pub fn run_basic_dynamic_function(store: &Store, compiler_name: &str, c: &mut Cr
     );
 }
 
-fn run_static_benchmarks(_c: &mut Criterion) {
+fn run_static_benchmarks(c: &mut Criterion) {
     #[cfg(feature = "llvm")]
     {
-        let store = Store::new(&JIT::new(wasmer_compiler_llvm::LLVM::new()).engine());
+        let store =
+            Store::new(&wasmer_engine_jit::JIT::new(wasmer_compiler_llvm::LLVM::new()).engine());
         run_basic_static_function(&store, "llvm", c);
     }
 
     #[cfg(feature = "cranelift")]
     {
-        let store = Store::new(&JIT::new(wasmer_compiler_cranelift::Cranelift::new()).engine());
+        let store = Store::new(
+            &wasmer_engine_jit::JIT::new(wasmer_compiler_cranelift::Cranelift::new()).engine(),
+        );
         run_basic_static_function(&store, "cranelift", c);
     }
 
     #[cfg(feature = "singlepass")]
     {
-        let store = Store::new(&JIT::new(wasmer_compiler_singlepass::Singlepass::new()).engine());
+        let store = Store::new(
+            &wasmer_engine_jit::JIT::new(wasmer_compiler_singlepass::Singlepass::new()).engine(),
+        );
         run_basic_static_function(&store, "singlepass", c);
     }
 }
 
-fn run_dynamic_benchmarks(_c: &mut Criterion) {
+fn run_dynamic_benchmarks(c: &mut Criterion) {
     #[cfg(feature = "llvm")]
     {
-        let store = Store::new(&JIT::new(wasmer_compiler_llvm::LLVM::new()).engine());
+        let store =
+            Store::new(&wasmer_engine_jit::JIT::new(wasmer_compiler_llvm::LLVM::new()).engine());
```

Comment:
Why is it necessary to use the full name here? The module (here `JIT`) is re-exported from `wasmer` as far as I know.

Reply:
Yeah, good call. Not sure why that was needed.

I added the change because `cargo bench` was actually not working before and it suggested that change. I can check again.